manifests/repo.pp

```diff
@@ -0,0 +1,26 @@
+class squid::repo
+{
+  if $::squid::manage_repo {
+    case $::osfamily {
+      'RedHat': {
+        case $::operatingsystemmajrelease {
+          '6','7': {
+            yumrepo { 'squid':
+              name     => 'Squid',
+              descr    => 'Squid repo for CentOS Linux - $basearch',
+              baseurl  => 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/',
```

Comment:
@elico Thanks again.

Should I be using 'http://www1.ngtech.co.il/repo/centos/$releasever/$basearch/' for all EL variants?  I noticed, for example, there is also http://www1.ngtech.co.il/repo/rhel/7/x86_64/ but the packages in there aren't as up to date.